src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/UnitTestingIncrementalAnalyzerProvider.cs

```diff
@@ -10,11 +11,23 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting
     internal class UnitTestingIncrementalAnalyzerProvider : IIncrementalAnalyzerProvider
     {
         private readonly IUnitTestingIncrementalAnalyzerProviderImplementation _incrementalAnalyzerProvider;
+        private IIncrementalAnalyzer _analyzer;
```

Comment:
Can you link to the implementation so I can review for potential long-lived state variables?